test/v1alpha1/retry_test.go

```diff
@@ -44,6 +48,9 @@ func TestTaskRunRetry(t *testing.T) {
 	knativetest.CleanupOnInterrupt(func() { tearDown(ctx, t, c, namespace) }, t.Logf)
 	defer tearDown(ctx, t, c, namespace)
 
+	// Skip if the "embedded-status" feature flag has any value but "full".
```

Comment:
Nope, that's still relevant in my mind - `v1alpha1` should, in theory, always be `full`. Though in practice because of how we have the type aliases set up, I guess you could mess around there. Eh, I'm fine either way.

Reply:
either way makes sense, but the comment here doesn't reflect the code.